scripts/update_all_products.pl

```diff
@@ -203,7 +207,9 @@
 	and (not $remove_team) and (not $remove_label) and (not $remove_nutrient)
 	and (not $mark_as_obsolete_since_date)
 	and (not $assign_categories_properties) and (not $restore_values_deleted_by_user) and not ($delete_debug_tags)
-	and (not $compute_codes) and (not $compute_carbon) and (not $check_quality) and (scalar @fields_to_update == 0) and (not $count) and (not $just_print_codes)) {
+	and (not $compute_codes) and (not $compute_carbon) and (not $check_quality) and (scalar @fields_to_update == 0) and (not $count) and (not $just_print_codes)
+	and (not $mongodb_to_mongodb)
```

Comment:
The purpose of this check is to make sure that the user specified something to update. mongodb to mongodb is more of an option, so specifying it should not prevent the check from verifying we'll actually update something.

Reply:
```suggestion
```